src/styles/thumbnail.css

```
@@ -49,7 +64,10 @@
   display: flex;
   flex-direction: column;
   gap: var(--thumbnail-desc-gap);
-  min-height: calc(var(--thumbnail-rate-height) + var(--thumbnail-title-height) + var(--thumbnail-desc-gap));
+  min-height: calc(
+    var(--thumbnail-rate-height) + var(--thumbnail-title-height) +
```

Stylelint (17.7.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)